### DISCUSSION TITLE
Why is this online meeting not published in the Meeting Guide App

### DISCUSSION THREAD

**Discussion by mikec-ncsd** [code4recovery/12-step-meeting-list#672] [Q&A] [answered]
I have an online only meeting that is not showing up in the Meeting Guide App:
https://ncsandiegoaa.org/meetings/7am-zoom-3rd-tradition-belly-up?tsml-day=any&tsml-query=belly

A live meeting with a specific address in the same city does show up in the Meeting Guide App:
https://ncsandiegoaa.org/meetings/3rd-tradition-belly-up-6?tsml-day=any&tsml-query=belly

Both of these meetings publish on our website using TMSL WP plugin.

Even though both of these meetings are in the same city, I suspect the Meeting Guide App is filtering the zoom meeting out of our region since it is on the border of our area.

**Comment by joshreisner** [comment, marked as answer]
@mikec-ncsd it looks like the meeting is entered correctly, my guess it’s an issue with an overlap with another area. 

we don’t manage the app, please reach out to appsupport@aa.org and include that url

**Comment by mikec-ncsd** [comment]
That did it. We were overlapping with another area. appsupport responded quickly.
